bgpd/bgp_zebra.c

```diff
@@ -1230,8 +1291,12 @@ void bgp_zebra_announce_table(struct bgp *bgp, afi_t afi, safi_t safi)
 	for (rn = bgp_table_top(table); rn; rn = bgp_route_next(rn))
 		for (ri = rn->info; ri; ri = ri->next)
 			if (CHECK_FLAG(ri->flags, BGP_INFO_SELECTED)
-			    && ri->type == ZEBRA_ROUTE_BGP
-			    && ri->sub_type == BGP_ROUTE_NORMAL)
+			    &&
+
```

Comment:
Bad style, please fix.

Reply:
I am not finding guidance on this at https://www.kernel.org/doc/html/latest/process/coding-style.html. Perhaps I have overlooked something or this URL is not the correct reference?